runners/flink/flink_runner.gradle

```diff
@@ -68,10 +68,15 @@ test {
   if (System.getProperty("beamSurefireArgline")) {
     jvmArgs System.getProperty("beamSurefireArgline")
   }
-}.onlyIf {
-  // TODO Running tests of all Flink versions in parallel can be too harsh on Jenkins memory
-  // Skip 1.6/1.7 tests for now, to avoid "Exit code 137", i.e. Jenkins host killing the Gradle test process
-  project.name != "beam-runners-flink-1.6" && project.name != "beam-runners-flink-1.7"
+  if (project.jenkins.isCIBuild) {
+    // TODO Running tests of all Flink versions in parallel can be too harsh on Jenkins memory
+    // Run them serially for now, to avoid "Exit code 137", i.e. Jenkins host killing the Gradle test process
+    if (project.name == "beam-runners-flink-1.6") {
+      dependsOn(":beam-runners-flink_2.11:test")
```

Comment:
Should we prefer `mustRunAfter`here? `dependsOn`will add a dependency, so if we intend to execute 'beam-runners-flink-1.6' that 'beam-runners-flink_2.11:test' will also be run, i.e. gradle will add the latter to the execution graph even if it was not requested.

In contrast, 'mustRunAfter' will only order the task, if *both* are present on the execution graph. If only the former is requested, the latter will not run.

Reply:
although this will require to specify the total order, i.e. declare all combinations

Reply:
Yes, that was my intention and I couldn't find another way of doing this. Thank you will update the PR.